src/Spout/Writer/Common/Sheet.php

```diff
@@ -33,6 +33,9 @@ class Sheet
     /** @var string Name of the sheet */
     protected $name;
 
+     /** @var bool isVisible visibility of the sheet */
```

Comment:
`/** @var bool Visibility of the sheet */` (removing the reference to the var name)